src/app/(settings)/settings/profile/components/Profile.tsx

```diff
@@ -59,10 +58,13 @@ const Profile = ({ loginUser }: SettingsProps) => {
         </div>
 
         <div className="space-y-2">
-          <Label>Address</Label>
-          <div className="border rounded-md p-2 bg-gray-50 text-xs">
-            {loginUser.address}
-          </div>
+          <Label htmlFor="address">Address</Label>
+          <textarea
```

Comment:
Hmm, let's talk about semantics here. If we need to display an address field, wouldn't `<address>` tag be the most appropriate?

Reply:
Asking only because text area is `readOnly`